CHANGELOG.md

```diff
@@ -54,6 +54,18 @@ Other Changes
 - Lepton was upgraded to the latest version (PR #349)
 - Made Object::print a const member function (PR #191)
 - Improved the testOptimization/OptimizationExample to reduce the runtime (PR #416)
+- Added shouldBeParallelIfPossible() override methods to all computationally
+expensive muscle force classes
```

Comment:
`shouldBeParallelIfPossible` is verbose but I am not sure it is informative. `computeInParallelWithOtherForces` may be more accurate?